malduck/procmem/procmem.py

```diff
@@ -11,7 +11,19 @@
 
 __all__ = ["ProcessMemory", "procmem"]
 
-ProcessMemoryBuffer = Union[bytes, bytearray, mmap.mmap]
+
+class MemoryBuffer(object):
```

Comment:
I think we don't need that as we stopped supporting Py2
```suggestion
class MemoryBuffer:
```